src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1885,89 +1901,80 @@ private static void validateChapterWiseTesting(Iterator<Node> itrChecks,
             Node config = null;
 
             try {
-                config = itrConfigs.next();
+                config = itrChecks.next();
             }
             catch (NoSuchElementException ignore) {
                 assertWithMessage(styleName + "_style.xml rule '" + ruleName + "' module '"
-                        + moduleName + "' is missing the config link: config").fail();
+                        + moduleName + "' is missing the config link").fail();
             }
 
             final String configUrl = config.getAttributes().getNamedItem("href")
                     .getTextContent();
 
-            final String expectedUrl = "https://github.com/search?q="
-                    + "path%3Asrc%2Fmain%2Fresources%20path%3A**%2F" + styleName
+            final String wholeConfigUrl = partialConfigUrl
```

Comment:
Keep this as `expectedUrl`. It makes sense to name these variables as expected/actual so it is easier to track which is which and not just temporary stuff.

Reply:
done